org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/doc/ApiDocGenerator.java

```diff
@@ -864,6 +873,169 @@ public MethodDescriptor withProduces(String produces) {
     }
   }
 
+  public static class MethodSignatureDescriptor implements Comparable<MethodSignatureDescriptor> {
+
+    private Method m_method;
+
```

Comment:
remove newline

Reply:
theoretically correct, but i based the formatting on the other descriptor classes. e.g. see MethodDescriptor.m_method